libpod/pod.go

```diff
@@ -128,6 +131,48 @@ func (p *Pod) Namespace() string {
 	return p.config.Namespace
 }
 
+// ResourceLim returns the cpuset resource limits for the pod
+func (p *Pod) ResourceLim() *specs.LinuxResources {
+	resCopy := &specs.LinuxResources{}
+	if err := JSONDeepCopy(p.config.InfraContainer.ResourceLimits, resCopy); err != nil {
+		return nil
+	}
+	if resCopy != nil && resCopy.CPU != nil {
+		return resCopy
+	}
+	pe := uint64(0)
+	q := int64(0)
+	c := ""
+	empty := &specs.LinuxResources{
+		CPU: &specs.LinuxCPU{Period: &pe, Quota: &q, Cpus: c},
```

Comment:
```suggestion
		CPU: &specs.LinuxCPU{},
```
nit, not need to populate the fields with go Zero-values.

Reply:
@jwhonce Period and Quota are pointers, does this affect access (with a nil pointer deref) if i don't initialize them?

Reply:
nevermind, just had to change a line that checked `*.CPU.Period`'s value to checking if `.CPU.Period != nil`